ui/app.yaml

```diff
@@ -12,7 +12,7 @@ handlers:
     Strict-Transport-Security: "max-age=31536000; includeSubDomains; preload"
     X-XSS-Protection: 1
     X-Content-Type-Options: "nosniff"
-    Content-Security-Policy: "default-src 'none'; frame-ancestors 'none'; report-uri /content-security-report"
+    Content-Security-Policy: "default-src 'none'; img-src 'self'; style-src 'unsafe-inline'; frame-ancestors 'self'; report-uri /content-security-report"
```

Comment:
I don't think so - the html that gets generated has the css directly defined in the style tag, that is why it is needed.

I did run these changes by @calbach before adding them. CH - would you mind taking a second look at the updated CSP?

Reply:
Yeah, this is OK. unsafe-inline is not ideal, but we have a similar requirement for rendering the notebook HTML previews; our hands are a bit tied since we don't have (or want to have) fine control over the generation of the HTML content.

Later we could try to tune this a bit by only allowing unsafe-inline on specific affected resources, but it won't make a difference currently as I believe this is the only HTML served under this file pattern (the pattern below is used for our main index.html, and has its own CSP)